Documentation/operations/troubleshooting.rst

```diff
@@ -277,25 +276,16 @@ command:
 
 .. code:: bash
 
-   hubble status --server localhost:4245
+   hubble status
```

Comment:
The page does not document that this command uses `localhost:4245` as default values. If users decide for whatever reason to change the address or port at the `port-forward` step, then `hubble status` will fail and they won't understand why.

Could we please add a brief subsection or note to mention the existence of these default values, and of the `--server` option? (The remark also applies to subsequent commit.)

Reply:
Discussed offline and the note is not super relevant. Users should look at the `--help` output to find the documentation for such options.

Reply:
We discussed this offline. I will add a note on the gsg with hubble guide which is linked from the troubleshooting guide.

Reply:
Just added this note:
![image](https://user-images.githubusercontent.com/1324157/116987974-0a3ee080-acd0-11eb-8b6e-534e144ff4a1.png)